modules/dashboard/test/TestPlan.md

```diff
@@ -2,3 +2,6 @@
    [Automate Test on Travis_CI]
 2. Check performance on the raisin bread dataset: make sure the dashboard page displays within 5 seconds or less.
    [Manual Test]
+3. Verify that if user has certain permission(s), the corresponding widget(s) will be displayed on the dashboard. [Automated Test]  
```

Comment:
This should go into more details. See  modules/candidate_profile/test/TestPlan.md for an example.

Reply:
changed / updated